client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -254,6 +254,10 @@ private void addDevOverrides(RequestBuilder fullRequest) {
                     return false;
                 }
                 if (((WebException)throwable).getStatus().code() >= 500) {
+
+                    // Log the error on every retry.
+                    LOG.info("Will retry because an error occurred", throwable);
```

Comment:
If I'm not mistaken, this will not log anything about the request, you probably want to log something like `String request = format("%s, headers: %s", fullReq.getFullUrl(), fullReq.getHeaders().entrySet());`, or is this already part of the caught WebException?

Reply:
It is my understanding that the caught WebException includes the response from the service and that is what we want to log.